### PR TITLE
First-Capstone

### PR DESCRIPTION
**Pull request by Gmanlove**
### **I did this capstone project by:**

- [ ] Creating a new repository for the project
- [ ] Creating a new branch (capstone1) for the project
- [ ] Checking out the Cindy shin template for the project and download some images that best fit the project
- [ ] Creating a Javascript file to design my hamburger bar and make the featured speaker section dynamic
- [ ] Ensuring no linter error
- [ ] Recording a video presenting the features of the project I built. which can be accessed by the link below.
(LOOM VIDEO) https://www.loom.com/share/bf2eb1a08afb4a9fbafd9b1f0612325c

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Join</a>
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Sponsors</a>
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Contact</a>
+                </li>
```
Comment (christianbiring1) [inline about.html]: - [ ] Kindly make sure when the user is on the about page, the link in the navigation bar is `home` to navigate on the home(`index`) page.
- [ ] Also, consider linking your link in the navigation bar with the corresponding section on the page. `i.e when the user clicks the link they are redirected to the exact section on the page(sponsors, program,...)`

File: style.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    <div class="image">
+                      <img src="${firstSpeaker[i].Image}" alt="speaker-image">
+                    </div>
+                    <div class="okwu">
+                    <h2>${firstSpeaker[i].name}</h2>
+                    <p><em>${firstSpeaker[i].title}</em></p>
+                    <div class="line2"></div>
+                    <p>${firstSpeaker[i].desc}</p>
+                    </div>
+                </div>
```
Comment (christianbiring1) [inline style.js:74]: [OPTIONAL]
- [ ] Kindly make sure on the mobile version, once the page is visited, the user can only see some of the `sponsors` and once the `SEE MORE` button is clicked, they can be able to see the remaining sponsor
`HINT: You may consider displaying half of the sponsor array elements at first and setting the other half hidden, and once the button is clicked all the elements are displayed on the UI. and the text content in the button change to SEE LESS`
image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,976 @@
...
+}
+
+.ul-items {
+  display: block;
+  flex-direction: row;
+  width: 100%;
+  margin-left: 42%;
+}
+
+.first-section {
```
Comment (christianbiring1) [inline style.css:224]: - [ ] Kindly make sure the first section on the about section is well centered as in the original template

Your design :point_down: 
Screenshot from 2023-02-09 22-26-32

File: style.css [hunk 4]
```diff
@@ -0,0 +1,976 @@
...
+.intro h3 {
+  font-size: 40px;
+  font-weight: bold;
+  color: #fff;
+}
+
+/* mobile view */
+
+@media screen and (max-width: 768px) {
+  .hamburger {
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure The CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around as it is the best practice and helps to reduce the code lines.
The media queries is supposed to be:
```css
@media screen and (min-width: 768px)
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,976 @@
...
+  color: #fff;
+}
+
+.ukwu-box {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 20px;
+  margin: 40px 0;
+  padding: 30px 0;
```
Comment (christianbiring1) [inline style.css:483]: - [ ] Kindly make sure you stick to the design as much as possible. please consider adding this overlay on these images on the `about` page as is in the design.
`Hint: You may add the linear gradient to the background if the image was added background image`
Your design :point_down: 
image

Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Gmanlove , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @Gmanlove , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Great pull request title and description :heavy_check_mark: :
- [ ] Professional README :heavy_check_mark: 
- [ ] All changes made:heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**